### PR TITLE
[READY] Create Kulumbegov AL

### PR DESCRIPTION
**Pull request by KulumbegovAI**
Student of eltech. 17 y.o. tryed java

### CODE REVIEW

**Comment by Toxa-man** [comment]
[READY] should be in request's title, not in comments section

File: Kulumbegov A.I. [hunk 1]
```diff
@@ -0,0 +1 @@
+Student of eltech. 17 y.o. tryed java
```
Comment (Toxa-man) [inline Kulumbegov A.I.]: This file should be created in your own directory, not in root
See other for example

Comment (Toxa-man) [reply Kulumbegov A.I.]: Still not fixed

File: Kulumbegov A.L./[READY].txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Student of eltech, 17 y.o. tryed Java
```
Comment (Toxa-man) [inline Kulumbegov A.L./[READY].txt]: There should be no dots in directory name
Also do not use pull request prefix in file names

**Comment by KulumbegovAI** [comment]
fixed

**Pull request merged**